docker/scripts/install-docker-compose-ubuntu.sh

```diff
@@ -0,0 +1,28 @@
+# Ask for the user password
+# Script only works if sudo caches the password for a few minutes
+sudo true
```

Comment:
It's not a good security practice to ask for user passwords within a script. It's better to check whether the script has sudo-permission, and if not, ask the user to re-run with sudo:

https://github.com/rhcs-dashboard/ceph-dev/blob/3aa188a5ed2efaa142b9bfe2088743a29b405e92/docker/scripts/install-docker-compose-centos-rhel.sh#L5-L9

Reply:
Thanks a lot. Let me work on this!